src/frontend/apps/impress/src/components/Icon.tsx

```diff
@@ -5,10 +5,19 @@ import { useCunninghamTheme } from '@/cunningham';
 
 type IconProps = TextType & {
   iconName: string;
+  isFilled?: boolean;
 };
-export const Icon = ({ iconName, ...textProps }: IconProps) => {
+export const Icon = ({ iconName, isFilled, ...textProps }: IconProps) => {
   return (
-    <Text $isMaterialIcon {...textProps}>
+    <Text
+      $isMaterialIcon={!isFilled}
+      {...textProps}
+      className={
+        isFilled
+          ? `material-icons-filled ${textProps.className}`
+          : textProps.className
+      }
+    >
```

Comment:
Could we change the type of `$isMaterialIcon` instead to be `boolean | "filled"` and add the className accordingly on the Text component ?